scripts/livepeer-core-dump-monitor

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This script monitors the CORE_DUMP_DIR directory (set in the Makefile)
+# and when new files are added it pops up a red notification and attempts
+# to use GDB to produce a stack trace.
+
+set -euo pipefail
+
+if [[ $* == *-j* ]]; then
+    echo '{
+  "desc": "Livepeer-packaged core dump monitor",
+  "name": "Livepeer core dump monitor",
+  "optional": {}
+}'
+    exit 0
+fi
+
+DIR="${CORE_DUMP_DIR:-}"
+if [[ "$DIR" == "" ]]; then
+  echo "No CORE_DUMP_DIR provided. Doing nothing forever."
+  exec tail -f /dev/null
+fi
+
+logOfChanges="/tmp/changes.log.csv" # Set your file name here.
```

Comment:
is it used ?

Reply:
Nice catch 51afff39f405751a12367171515554a7f32ec3b6